salt/states/group.py

```diff
@@ -72,21 +72,24 @@ def _changes(name,
             delusers = [salt.utils.win_functions.get_sam_name(user).lower() for user in delusers]
 
     change = {}
-    ret = {}
     if gid:
         try:
             gid = int(gid)
             if lgrp['gid'] != gid:
                 change['gid'] = gid
         except (TypeError, ValueError):
-            ret['result'] = False
-            ret['comment'] = 'Invalid gid'
-            return ret
+            return {
```

Comment:
I'm not following the need for this change. Could you clarify, please?

Reply:
Eh, similar to the elif: vs. if:continue change below, I should probably have held off until offering this for the develop branch. I'll revert both so that this is only a fix for the reported issue.

My logic for dropping ret={} was that ret is used nowhere else and logically wouldn't pop up. There's an empty dict created, manipulated, and returned instead of just returning the dict that needs to be returned.